cmake/common/eprosima_libraries.cmake

```diff
@@ -55,7 +55,10 @@ macro(eprosima_find_package package)
                 foreach(opt_ ${FIND_OPTIONS})
                     set(${opt_} ON)
                 endforeach()
+                set(EPROSIMA_INSTALLER_TEMP ${EPROSIMA_INSTALLER})
```

Comment:
Please, add a comment to explain why we decided to do this. And update our subtree cmake-common

Reply:
Added